README.md

```diff
@@ -128,6 +128,31 @@ def function(request):
 This function will catch the `ZeroDivisionError` and return a different
 response instead.
 
+### Quickstart: With Pub/Sub emulator
+
+1. Start Functions Framework in your local development server listening on port 8082:
+   ```sh
+    functions-framework --target=hello --debug --port=8082
+   ```
+1. Start the Pub/Sub emulator.
+   ```sh
+    export PUBSUB_PROJECT_ID=abc
+    gcloud beta emulators pubsub start \
+        --project=$PUBSUB_PROJECT_ID \
+        --host-port=localhost:8080
+   ```
+1. Create a Pub/Sub topic and attach a push subscription to the topic, using `http://localhost:8082` as its push endpoint. Publish some messages to the topic. Observe your function getting triggered by Pub/Sub messages.
```

Comment:
This step seems very large. Can we split it up into multiple steps?

Reply:
I put these steps together because there are multiple terminals used in this quickstart. All these steps should be run in the same terminal. And they all do Pub/Sub-related things. I vote we keep them together.